src/controllers/organizationController.ts

```diff
@@ -196,21 +196,34 @@ export default class OrganizationController {
             );
           }
           const organizationInstance = maybeOrganizationInstance.value;
-          // TODO:
-          //  the documents must be digitally signed before being sent via email.
-          //  @see https://www.pivotaltracker.com/story/show/169726141
+          const unsignedContractPath = `./documents/${organizationInstance.ipaCode}/contract.pdf`;
+          const signedContractPath = `./documents/${organizationInstance.ipaCode}/signed-contract.pdf`;
+          const unsignedMandatePath = `./documents/${
+            organizationInstance.ipaCode
+          }/mandate-${user.fiscalCode.toLocaleLowerCase()}.pdf`;
+          const signedMandatePath = `./documents/${
+            organizationInstance.ipaCode
+          }/signed-mandate-${user.fiscalCode.toLocaleLowerCase()}.pdf`;
+
+          const arrayOfMaybeError = await Promise.all([
+            this.createSignedDocument(unsignedContractPath, signedContractPath),
+            this.createSignedDocument(unsignedMandatePath, signedMandatePath)
+          ]);
+          if (arrayOfMaybeError.some(isSome)) {
```

Comment:
is it possible to keep and display the error message here ? ie.

`const error = arrayOfMaybeError.filter(isSome)...;`